src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -32,6 +32,12 @@ public abstract partial class TestKitBase
         public object FishForMessage(Predicate<object> isMessage, TimeSpan? max = null, string hint = "")
         {
             return FishForMessage<object>(isMessage, max, hint);
```

Comment:
Why isn't this sync over async?

Reply:
It is but not directly - I will change that to call `async` version of itself!